ci/docker/common.yaml

```diff
@@ -46,3 +46,6 @@ packages:
     # Force git as non-buildable to allow deprecated versions in environments
     # https://github.com/spack/spack/pull/30040
     buildable: false
+  pika:
+    require:
+      - '@git.release-0.30.X=0.30.1'
```

Comment:
Removing this since we've made one round through CI with what became pika 0.30.1.
```suggestion
```